book/website/collaboration/maintain-review/maintain-review-checklist.md

```diff
@@ -0,0 +1,15 @@
+(cl-maintain-review-checklist)=
```

Comment:
```suggestion
(cl-maintain-review-resources)=
```

Reply:
Per the chapter template, there should be separate `Checklist` and `Resources` subchapters

Reply:
Since we don't have a checklist, we can just name this `Resources`

Reply:
So we should also change the filenames, too?

Reply:
Oh yes, we should. Thanks for catching that!

Reply:
Alright, made a commit to change filenames and contents.